deployment/monocular/templates/ui-deployment.yaml

```diff
@@ -11,7 +11,7 @@ spec:
       labels:
         app: {{ template "fullname" . }}-ui
       annotations:
-        checksum/config: {{ print .Values.api.config | sha256sum }}
+        checksum/config: {{ include (print $.Template.BasePath "/ui-config.yaml") . | sha256sum }}
```

Comment:
What's the difference here? That you are actually now rendering the content?

Reply:
Yes, honestly I'm not sure why it was this way before, the Helm documentation recommends doing it this way: https://github.com/helm/helm/blob/master/docs/charts_tips_and_tricks.md#automatically-roll-deployments-when-configmaps-or-secrets-change